src/test/java/uk/gov/hmcts/ccd/domain/types/CaseDataValidatorTest.java

```diff
@@ -177,8 +190,9 @@ public void unknownType() throws Exception {
                 "}";
         final Map<String, JsonNode> values = MAPPER.readValue(data, new TypeReference<HashMap<String, JsonNode>>() {
         });
-        final List<ValidationResult> result = caseDataValidator.validate(values, caseFields);
-        assertEquals(result.toString(), 1, result.size());
+        final ValidationContext validationContext = getValidationContext(values);
+        final List<ValidationResult> result = caseDataValidator.validate(validationContext);
+        assertEquals(result.toString(), 2, result.size());
```

Comment:
Just wondering..why is this now 2 - is a change expected in the number of errors from this test? (Sorry, bit hard to understand the context since not many changes were made around this and it's mostly old code 😄 )